tasks/phylogenetic_inference/task_snp_dists.wdl

```diff
@@ -10,12 +10,97 @@ task snp_dists {
     date | tee DATE
     snp-dists -v | tee VERSION
 
-    snp-dists ~{alignment} > ~{cluster_name}_snp_distance_matrix.tsv
+    # create snp-dists matrix file
+    snp-dists ~{alignment} > snp-dists-matrix.tsv 
+
+    # create oredered snp-dists molten file 
+    snp-dists -m ~{alignment} | awk '{print $NF,$0}' | sort -n | cut -f2- -d' ' > snp-dists-molten-ordered.tsv 
+
+    # create list of isolates in order of SNP-dists 
+    cut -f2 snp-dists-molten-ordered.tsv | awk '!seen[$0]++' > ordered-isolates.tsv
+
+    python <<CODE
+    # This script is based on Logan Fink's https://github.com/StaPH-B/CDPHE/blob/master/ordered_pairwise_matrix_generator_1.0.py
+    # which takes in an ordered file and then returns a pairwise matrix based on that order.
+    import sys, os
+    cwd = os.getcwd()
+
+    #Create a pairwise dictionary with all values possible, and create a list of the isolates
+    pairwiseDict = {}
+    isolates = []
+    f = open("snp-dists-molten-ordered.tsv", 'r')
+    flines  = f.readlines()
+    for line in flines:
+        print line
```

Comment:
should you rework this to be python3 since python2 isn't supported?

Reply:
I was thinking the same, but I want to keep things within this task and it doesn't look like python3 is within this [snp-dists image](https://github.com/StaPH-B/docker-builds/blob/master/snp-dists/0.8.2/Dockerfile)